a/ampifitia.md

```diff
@@ -0,0 +1,9 @@
+# Ampifitia
+
+![Ampifitia](ampi.JPG)
+
+This artist started in 2002 and became famous in 2013.It is a christian band.
+
+**Genre:** Country and slow
+
+**Artist homepage:** [Ampifitia](www.facebook.com )
```

Comment:
Does Ampifitia have a facebook page? This URL you've used is the facebook homepage. That's not very useful!